src/sql/workbench/services/notebook/browser/notebookService.ts

```diff
@@ -44,6 +46,17 @@ export enum NavigationProviders {
 
 export const unsavedBooksContextKey = 'unsavedBooks';
 
+export interface INotebookOpenOptions {
```

Comment:
Isn't this the same as INotebookShowOptions? Why do we need a new interface here?